spark8t/resources/templates/role_yaml.tmpl

```diff
@@ -2,7 +2,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   labels:
-    app.kubernetes.io/managed-by: spark-client
+    app.kubernetes.io/managed-by: spark8t
```

Comment:
Let me know if this was the right thing to do

Reply:
Yes, that's fine!

Reply:
Cool, thx.